bike_kollective/lib/data/provider/active_user.dart

```diff
@@ -31,12 +35,15 @@ class ActiveUserNotifier extends StateNotifier<UserModel?> {
   Future<UserModel> signIn(SignInMethod method) async {
```

Comment:
Rename method to signInWithGoogle and remove the parameter SignInMethod and remove the definition of SignInMethod (since it seems like we aren't using it given you have a separate function for email).